pkg/user/password_reset.go

```diff
@@ -0,0 +1,42 @@
+package user
+
+import (
+	"time"
+
+	"github.com/dgryski/trifles/uuid"
```

Comment:
Use `github.com/google/uuid` (use a well-maintained library for security reasons)